### PR TITLE
new feature map static

### PR DESCRIPTION
**Pull request by breitembach**
i will start to create new this feature

### CODE REVIEW

**Comment by lejard-h** [comment]
Nothing commited. Is it normal ?

**Comment by breitembach** [comment]
done!

File: lib/src/staticmap.dart [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  
+  Path({
+    this.color,
+    this.enc,
+  });
+
+  @override
+  String toString() {
+    String stringPath = 'color:$color|enc:$enc';
+    // if(markers?.isNotEmpty ?? false) {
```
Comment (lejard-h) [inline lib/src/staticmap.dart:17]: remove unnecessary commented line

File: lib/src/staticmap.dart [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    this.scale = 'false',
+    this.path,
+    this.mapType = 'roadmap'
+  }): assert(
+    size.contains('x'),
+  ) {
+
+  }
+
+  // Image getImage() {
```
Comment (lejard-h) [inline lib/src/staticmap.dart:53]: commented lines and seems related to Flutter

File: lib/src/staticmap.dart [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    this.markers,
+    this.zoom = '', 
+    this.size = '580x267',
+    this.center = '',
+    this.scale = 'false',
+    this.path,
+    this.mapType = 'roadmap'
+  }): assert(
+    size.contains('x'),
+  ) {
```
Comment (lejard-h) [inline lib/src/staticmap.dart:49]: don't open braces in constructor if nothing in it

**Comment by breitembach** [comment]
Ok

**Comment by lejard-h** [comment]
Thanks ! and sorry it was long ...

**Pull request merged**